tendrl/monitoring_integration/graphite/graphite_utils.py

```diff
@@ -88,6 +88,8 @@ def _add_metrics(objects, obj_name, metric, resource):
                     try:
                         if key == "details":
                             continue
+                        if obj == "GeoReplicationSession":
+                            obj = "geo_rep_session"
```

Comment:
while pushing into graphite only "GeoReplicationSession" is changed to "geo_rep_session"

Reply:
@GowthamShanmugam , can you get some details here GeoReplicationSession vs geo_rep_session and what we are trying to do here?

Reply:
in grafana volume dashbaord json we are actually expecting geo_rep_session count metrics like 
`tendrl.names.$cluster_id.volumes.$volume_name.geo_rep_session`

So while pushing geo_replication_session count I am forming metrics with a name "geo_rep_session", then only grafana can fetch detail from graphite.


GeoReplicationSession is a class name which is present under commons objects.

Reply:
Here I am changing "geo_rep_session" is only for a match with dashboard json metrics. otherwise, all geo_rep details are pushed in the name GeoRepSession in graphite and grafana will except name geo_rep_session in metrics so data won't be displayed.

Reply:
ok